examples/step-98/step-98.cc

```diff
@@ -0,0 +1,974 @@
+   *
+   * Calls clear on the base class and the `inverse` member.
+   */
+  template <int dim, int fe_degree, typename number>
+  void LaplacePatchSmoother<dim, fe_degree, number>::clear()
+  {
+    BaseType::clear();
+  }
+
+  /**
+   * @brief Local apply implementation.
+   *
+   * This function performs one smoother step (forward or backward Gauss-Seidel
+   * type iteration over patches) for a given range of patches. It calculates
+   * the local residual on each patch, applies the inverse of the local patch
+   * operator to compute a correction, and updates the global solution vector.
+   *
+   * Uses  FEPatchEvaluation to manage data transfer between
+   * global vectors and local patch vectors. Iterates through the specified
+   * patch range, computes the local residual, applies the patch inverse, and
+   * updates the global solution vector. At the  moment, only do_forward==true
+   * is supported by PatchStorage that manages the loop.
+   */
+  template <int dim, int fe_degree, typename number>
+  void LaplacePatchSmoother<dim, fe_degree, number>::local_apply(
+    const PatchStorageType &patch_storage,
+    VectorType             &dst, // Global solution vector (input/output)
+    const VectorType       &src, // Global right-hand side vector (input)
+    const typename PatchStorageType::PatchRange
+               &patch_range,      // Range of patches to process
+    const bool &do_forward) const // Direction of iteration
+  {
+    // Type alias for the FEPatchEvaluation, which handles operations across
+    // the cells within a patch, including data gathering and scattering.
+    // Moving values between patches vector and individual cell vectors, handled
+    // by the last template parameter, is crucial for efficiency. For the
+    // reference, all operations with distributor take around 10% of smoothing
+    // time, that is a comparable amount of time to inverse application via
+    // TensorProductMatrixSymmetricSum. If the residual is also  computed via
+    // TensorProductMatrixSymmetricSum this fraction is significantly higher.
+    //
+    // Micro-benchmarking can help determine the optimal method for
+    // distributing DoFs, as performance depends on several factors. For
+    // instance, with Clang, lookup tables are highly efficient; the overhead
+    // of copying data between cell vectors and patch vectors is minimal,
+    // comparable to a standard copy operation of the same data size. With GCC,
+    // an index-computing implementation (not included here) outperforms lookup
+    // tables, though it's still significantly slower than Clang. Using
+    // `DistributorLookup` is crucial for efficiency, as it provides the
+    // mapping between patch DoFs and cell DoFs.
+    using PatchEval =
+      FEPatchEvaluation<FEEval,
+                        PatchDistributors::DistributorLookup<dim, fe_degree>>;
+
+
+
+    // Instantiate the FEPatchEvaluation object. It takes the patch storage
```

Comment:
This seems like a slight inconsistency, I suggest to use (and as I said above, it might be helpful to make descriptions of more places with a focus on the big picture.
```suggestion
    // Set up the FEPatchEvaluation object. It takes the patch storage
```